src/passes/StripEH.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 WebAssembly Community Group participants
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//
+// Removes all EH instructions and tags. Removes catch blocks and converts
+// 'throw's into 'unreachable's. Any exception thrown will crash the program as
+// they are now traps.
+//
+
+#include <ir/drop.h>
+#include <ir/utils.h>
+
+namespace wasm {
+
+namespace {
+
+struct StripEHImpl : public WalkerPass<PostWalker<StripEHImpl>> {
+  bool refinalize = false;
+
+  bool isFunctionParallel() override { return true; }
```

Comment:
```suggestion
  bool isFunctionParallel() override { return true; }

  bool refinalize = false;
```
(as a convention we always put `isFunctionParallel` first)